drafter.gyp

```diff
@@ -428,6 +428,7 @@
       "conditions" : [
         [ 'libdrafter_type=="static_library"', { 'defines' : [ 'DRAFTER_BUILD_STATIC' ] }],
       ],
+      "defines": ["LOGGING"],
```

Comment:
I don't think this should be here.

Reply:
Why?

Reply:
Wrong message at wrong place. Ignore it.